mlir/lib/Target/Cpp/TranslateToCpp.cpp

```diff
@@ -774,6 +774,41 @@ static LogicalResult printOperation(CppEmitter &emitter,
   return printBinaryOperation(emitter, operation, "||");
 }
 
+static LogicalResult printOperation(CppEmitter &emitter,
```

Comment:
Move this function up, directly below the `cf.cond_br` function.

Reply:
Fixed.